osu.Game/Beatmaps/WorkingBeatmapCache.cs

```diff
@@ -141,6 +141,9 @@ protected override IBeatmap GetBeatmap()
                 try
                 {
                     string fileStorePath = BeatmapSetInfo.GetPathForFile(BeatmapInfo.Path);
+
+                    // TODO: check validity of file
```

Comment:
Any work remaining here to do, or is this best-effort for now?

Reply:
I plan on adding something here in the future. This PR isn't making the situation any worse so I figured I'd let that be for now.